packages/nodejs/src/tracer.ts

```diff
@@ -93,6 +93,21 @@ export class BaseTracer implements Tracer {
     return activeRootSpan
   }
 
+  /**
+   * Sends an error in a newly created `RootSpan` that will be closed after
+   * the given error is added to it.
+   *
+   * The created `RootSpan` is passed as the single argument to the given function.
+   * This allows you to add arbitrary metadata to it.
+   */
+  public sendError<T>(error: Error, fn: (s: NodeSpan) => T): void {
+    const rootSpan = new RootSpan()
+
+    rootSpan.setError(error)
+    fn(rootSpan)
```

Comment:
You'll need to check if `fn` is given here as well. If I don't pass it it gives me this error:

```
TypeError: fn is not a function
    at BaseTracer.sendError (/integration/packages/nodejs/dist/tracer.js:73:9)
    at /app/app.js:31:22
    at Layer.handle [as handle_request] (/app/node_modules/express/lib/router/layer.js:95:5)
    at next (/app/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/app/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/app/node_modules/express/lib/router/layer.js:95:5)
    at /app/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/app/node_modules/express/lib/router/index.js:335:12)
    at next (/app/node_modules/express/lib/router/index.js:275:10)
    at urlencodedParser (/app/node_modules/body-parser/lib/types/urlencoded.js:91:7)
```

Reply:
Included in: appsignal/appsignal-javascript#511. Once released and updated here, it should work.